app/src/main/java/mobile/programming/musicApp/dto/Instrument.kt

```diff
@@ -1,11 +1,11 @@
-package mobile.programming.musicapp.dto
+package mobile.programming.musicApp.dto
```

Comment:
Here, I have also changed the multi-name package name to camelcase.

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase